docs/threading.md

```diff
@@ -1,8 +1,10 @@
-Threads in libgit2
+Threading in libgit2
 ==================
 
-You may safely use any libgit2 object from any thread, though there
-may be issues depending on the cryptographic libraries libgit2 or its
+Unless otherwise specified, libgit2 objects cannot be safely accessed by
+multiple threads.
```

Comment:
Let's clarify further what this means.  eg:

> Unless otherwise specified, libgit2 objects cannot be safely accessed by multiple threads simultaneously.